interfaces/certificate_transfer/v1/schema.py

```diff
@@ -0,0 +1,34 @@
+"""This file defines the schemas for the provider and requirer sides of the `certificate_transfer` interface.
+It exposes two interface_tester.schema_base.DataBagSchema subclasses called:
+- ProviderSchema
+- RequirerSchema
+Examples:
+    ProviderSchema:
+        unit: {
+            "certificates": [
+                "-----BEGIN CERTIFICATE----- ...",
+                "-----BEGIN CERTIFICATE----- ..."
+            ]
+        }
+        app: <empty>
+    RequirerSchema:
+        unit: <empty>
+        app:  <empty>
+"""
+from pydantic import BaseModel, Field
+from typing import Set
+from interface_tester.schema_base import DataBagSchema
+
+class CertificateTransferProviderAppData(BaseModel):
+    certificates: Set[str]= Field(
+        description="The set of certificates that will be transferred to a requirer"
+    )
+
+class ProviderSchema(DataBagSchema):
+    """Provider schema for certificate_transfer."""
+    unit: CertificateTransferProviderAppData
```

Comment:
Why at the unit level? Wouldn't it make more sense for these to be app-level?

Reply:
it may be. I made it unit level because the previous object was also placed in the unit databag.

Reply:
we now use the app databag